libunwind/src/Unwind-wasm.c

```diff
@@ -10,14 +10,11 @@
 //
 //===----------------------------------------------------------------------===//
 
+#if __STDC_VERSION__ < 202311L
```

Comment:
This change seems unrelated to the rest of the patch.

Reply:
stdbool.h removed in C23